standard/expressions.md

```diff
@@ -2912,6 +2912,34 @@ These are the same transformations applied in [§6.4.3](lexical-structure.md#643
 
 An *anonymous_method_expression* is one of two ways of defining an anonymous function. These are further described in [§11.16](expressions.md#1116-anonymous-function-expressions).
 
+### §throw-expression-operator-new-clause The throw expression operator
```

Comment:
If `throw` is now an operator, does it need to be listed in 11.4.2?

Reply:
My first thought is that `throw` isn't an operator:

```suggestion
### §throw-expression-operator-new-clause The throw expression
```